controllers/articles-controllers.js

```diff
@@ -10,7 +10,7 @@ function getArticleById(request, response, next) {
   const articleId = request.params.article_id;
   selectArticleById(articleId)
     .then((results) => {
-      response.status(200).send({ article: results.rows });
+      response.status(200).send({ article: results.rows[0] });
```

Comment:
Ideally, we'd access the object from the array down in the model, as it's data manipulation which is the model's job - i know this is really picky